benchmarks/src/main/scala/akka.kafka.benchmarks/KafkaConsumerBenchmarks.scala

```diff
@@ -71,6 +71,8 @@ object KafkaConsumerBenchmarks extends LazyLogging {
 
     var lastProcessedOffset = 0L
     var accumulatedMsgCount = 0L
+    // TODO this is actually running single threaded, commitAsync is called on the poll thread.
+    //      It might not be waiting as it should for the commits to be completed?
     val semaphore = new Semaphore(1)
```

Comment:
well, I would put it in another way. If the `acquire()` ever blocks, then the only way to it to get unblocked would be that a previous commit callback is called, calling `release()`. But that can only happen in a `poll()`, which we will not call because the loop is blocked.